### PR TITLE
init

### PR DESCRIPTION
**Pull request by louis-nwosu**

### CODE REVIEW

File: src/components/home/productCard.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { Card } from "antd";
+import { notification } from "antd";
```
Comment (ChidiOkpala) [inline src/components/home/productCard.tsx:3]: Combine the import in line 2 and 3

**Review by ChidiOkpala — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 2]
```diff
@@ -1,24 +1,20 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+//import the router
+import { BrowserRouter, Switch, Route } from "react-router-dom";
```
[the remark this answers is not in the document]
Comment (louis-nwosu) [reply src/App.tsx:2]: alright...i'll change up everything to match the teams standards

File: src/components/cart/cartCard.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+import { notification } from "antd";
+
+interface ProductProps {
+  title: string;
+  category: string;
+  description: string;
+  image: string;
+  id: number;
+  price: number;
+  fn: (id: number) => void;
```
[the remark this answers is not in the document]
Comment (louis-nwosu) [reply src/components/cart/cartCard.tsx:12]: noted

File: src/components/cart/cartCard.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  category,
+  description,
+  image,
+  price,
+  id,
+  fn,
+}) => {
+  const OpenNotification = (placement: any) => {
+    notification.info({
+      message: ``,
```
[the remark this answers is not in the document]
Comment (louis-nwosu) [reply src/components/cart/cartCard.tsx:28]: noted...i'll keep that in mind

File: src/components/cart/cartCard.tsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+      placement,
+    });
+  };
+  return (
+    <React.Fragment>
+      <Card
+        cover={
+          <img
+            alt="product"
+            src={`${image}`}
```
[the remark this answers is not in the document]
Comment (louis-nwosu) [reply src/components/cart/cartCard.tsx:46]: i tried to but it was giving a weird error message...i'll look into it again

File: src/components/cart/cartCard.tsx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+  };
+  const OpenNotification2 = (placement: any) => {
+    notification.info({
+      message: ``,
+      description: "deleted from cart",
+      placement,
+    });
+  };
+  return (
+    <React.Fragment>
```
[the remark this answers is not in the document]
Comment (louis-nwosu) [reply src/components/cart/cartCard.tsx:41]: okay....the fragment is only a matter of personal preference..but i'll switch up to match teams standards

File: src/components/cart/cartDisplay.tsx [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+const styles = {
+  container: {
+    margin: "7%",
+  },
+};
+
+const DisplayCartItems: React.FC<{}> = () => {
+  const cartArr = useSelector((state: any) => state.cart);
+  console.log(cartArr);
```
[the remark this answers is not in the document]
Comment (louis-nwosu) [reply src/components/cart/cartDisplay.tsx:15]: lol....noted..sorry about that

File: src/components/home/footer.jsx [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+        <Col>
+          <h3 style={{color: '#fff'}}>Gotahia express.</h3>
+          <p style={{color: '#fff', fontWeight: 'bold'}}> made by louis </p>
+        </Col>
+      </Row>
+    </div>
+  );
+};
+
+export default Footer;
```
[the remark this answers is not in the document]
Comment (louis-nwosu) [reply src/components/home/footer.jsx:27]: i see...okay

File: src/components/home/productCard.tsx (hunk 1 above)
Comment (louis-nwosu) [reply src/components/home/productCard.tsx:3]: noted...i'll change that right away

**Pull request merged**